Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateMediaStreamAVFObjC.mm

```diff
@@ -1280,16 +1281,17 @@ static inline CGAffineTransform videoTransformationMatrix(VideoFrame& videoFrame
 
     m_storedBounds = sampleBufferDisplayLayer->rootLayer().bounds;
     m_storedBounds->size = size;
-    sampleBufferDisplayLayer->updateBoundsAndPosition(*m_storedBounds, WTFMove(fence));
+    sampleBufferDisplayLayer->updateBoundsAndPosition(*m_storedBounds, WTFMove(fence.sendRight));
 }
 
-void MediaPlayerPrivateMediaStreamAVFObjC::requestHostingContextID(LayerHostingContextIDCallback&& callback)
+void MediaPlayerPrivateMediaStreamAVFObjC::requestHostingContext(LayerHostingContextCallback&& callback)
 {
-    if (auto contextID = hostingContextID()) {
-        callback(contextID);
+    auto context = hostingContext();
+    if (hostingContext().contextID) {
```

Comment:
Why aren't we using `context` here instead of `hostingContext()`?

Reply:
Ah, good catch, fixed in latest patch!